### DISCUSSION TITLE
Popup/settings tab upon launch if new/changed settings detected

### DISCUSSION THREAD

**Discussion by i-liek-katz** [vladmandic/sdnext#1649] [Ideas]
Unsure if necessary, but if a user updates and there are detected changes in settings, or newly added settings (or removed settings?), a popup or defaulting to settings tab (maybe with a big printed not-popup message saying to click apply changes?) to try and push for changes being applied could be good, but I don't know if it's actually necessary or not. I typically kinda do this myself, and wondered if not doing this while updating could potentially cause issues, then felt like throwing the idea.

**Comment by vladmandic** [comment]
you already have visual indicator in settings for newly introduced settings, that was added about two weeks ago. i feel popup would be far too much.

**Reply by i-liek-katz** [reply]
Yea, I'm aware of it. I like to frequently apply settings after a git pull, but only made the idea in case there are issues if a user never actually applies settings. Unsure how it's handled, if all of that is already set or not, and the apply settings would just save it to the configs or something.

**Reply by vladmandic** [reply]
If it's not saved, it will use default. Apply just saves it to config.ui.
